CONTRIBUTING.rst

```diff
@@ -44,7 +44,7 @@ You'll have to create a development environment to hack on
    $ cd hack-on-google-cloud-python
    # Configure remotes such that you can pull changes from the google-cloud-python
    # repository into your local repository.
-   $ git remote add upstream https://github.com:GoogleCloudPlatform/google-cloud-python
+   $ git remote add upstream https://github.com/GoogleCloudPlatform/google-cloud-python.git
```

Comment:
I'd guess the goal was

```
git@github.com:GoogleCloudPlatform/google-cloud-python.git
```

This is "better" for maintainers with push rights that use SSH creds to push to GitHub